caffe2/ideep/operators/pool_op.cc

```diff
@@ -33,15 +35,18 @@ class IDEEPPoolOp final : public IDEEPConvPoolOpBase {
     auto& X = Input(INPUT);
     auto* Y = Output(OUTPUT);
     auto Y_dims = CalcOutputDims(X, X.get_dim(1));
+    mkldnn::prop_kind pk = training_mode_ ?
+      mkldnn::prop_kind::forward_training : mkldnn::prop_kind::forward_inference;
 
     ideep::pooling_forward::compute(X, Y_dims, *Y,
-        stride_, kernel_, pad_tl(), pad_br(), algo_);
```

Comment:
Just curious, what's the previous default that was causing the slowdown?

Reply:
@yinghai The default setting is training...

Reply:
OK...